### PR TITLE
Added "WIP" to README

### PR DESCRIPTION
**Pull request by RayanDharia**
The current website is a work-in-progress.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # rayandharia.github.io
-Rayan's Personal Website
+Rayan's Personal Website WIP
```
Comment (harrisonstark) [inline README.md:2]: Nit: Why did you add WIP and not W.I.P.?

Comment (RayanDharia) [reply README.md:2]: WIP is standard acronym

**Review by harrisonstark — APPROVED** [review]
